website/static/js/fangorn.js

```diff
@@ -1013,6 +996,491 @@ function scrollToFile(fileID) {
     }
 }
 
+// A fangorn-styled button; addons can reuse this
+var FGButton = {
+    controller: function(args) {
+        var noop = function() {};
+    },
+    view: function(ctrl, args, children) {
+        var extraCSS = args.className || '';
+        var tooltipText = args.tooltip;
+        var iconCSS = args.icon;
+        return m('div', {
+            className: 'fangorn-toolbar-icon ' + extraCSS,
+            onclick: args.onclick,
+            'data-toggle': tooltipText ? 'tooltip' : '',
+            'data-placement' : 'bottom',
+            'title':  tooltipText}, [
+            m('i', {className: iconCSS}),
+            m('span.hidden-xs', children)
+        ]);
+    }
+}
+
+var FGInput = {
+    controller : function(args) {
+        var noop = function() {};
```

Comment:
See above; this can be removed; noop only needs to be defined once in this module.